lib/pbench/server/api/resources/query_apis/controllers_list.py

```diff
@@ -130,6 +131,22 @@ def postprocess(self, es_json: Dict[AnyStr, Any]) -> Dict[AnyStr, Any]:
         ]
         """
         controllers = []
+        # If there are no matches for the user, controller name,
+        # and time range, return the empty list rather than failing.
+        # Note that we can't check the length of ["hits"]["hits"]
+        # because we've told Elasticsearch to return only aggregations,
+        # not source documents.
+        try:
+            count = es_json["hits"]["total"]["value"]
+            if int(count) == 0:
+                self.logger.warning("No data returned by Elasticsearch")
+                return jsonify(controllers)
+        except KeyError as e:
+            raise PostprocessError(
+                f"Can't find Elasticsearch match data {e} in {es_json}"
+            )
+        except ValueError as e:
+            raise PostprocessError(f"Elasticsearch hit count {count} value: {e}")
```

Comment:
It strikes me that you should "decorate" `{count}` somehow, here -- that is, if we couldn't convert it to an integer (which is, I presume, the cause of us seeing this message), then we might want to make clear what blank characters or other junk are in it.